### PR TITLE
Update docstrings of torch.nn.modules.activation.MultiheadAttention

### PR DESCRIPTION
**Pull request by Kyeongpil**
- Add the link to the original paper (Attention is All You Need)
- Fix indentation

### CODE REVIEW

File: torch/nn/modules/activation.py [hunk 1]
```diff
@@ -831,7 +831,7 @@ def extra_repr(self) -> str:
 class MultiheadAttention(Module):
     r"""Allows the model to jointly attend to information
     from different representation subspaces.
-    See reference: Attention Is All You Need
+    See reference: Attention Is All You Need(https://arxiv.org/abs/1706.03762)
```
Comment (heitorschueroff) [inline torch/nn/modules/activation.py:834]: See \`Attention Is All You Need \<https://arxiv.org/abs/1706.03762>`_

**Review by heitorschueroff — COMMENTED** [review]
Thanks for the PR. Just a small fix and it's good to go.

File: torch/nn/modules/activation.py (hunk 1 above)
Comment (Kyeongpil) [reply torch/nn/modules/activation.py]: Thank you for reviewing my PR. I updated the reference in the docstring.

**Comment by Kyeongpil** [comment]
@heitorschueroff
Actually, as shown in the picture, I think the format or indentation of "Inputs" and "Outputs" need to be changed. Because they are parent items of other items.
For example, query, key, value, key_padding_mask, and attn_mask are child items of "Inputs".
Do you have any idea?
image

**Comment by Kyeongpil** [comment]
@heitorschueroff Could you review my changes?

**Comment by heitorschueroff** [comment]
> @heitorschueroff
> Actually, as shown in the picture, I think the formats or indentations of "Inputs" and "Outputs" need to be changed. Because they are parent items of other items.
> For example, query, key, value, key_padding_mask, and attn_mask are child items of "Inputs".
> Do you have any idea?
> image

Good catch, could you try indenting the bullets under Inputs and Outputs one extra level?

**Review by heitorschueroff — APPROVED** [review]
Great, thank you for making the changes. Let me know if you'd like to fix the indentation of the shape portion as well, otherwise I can just merge this.

**Comment by Kyeongpil** [comment]
@heitorschueroff I tried to fix the indentation, but I don't know how to modify the docs while conforming to the sphinx (or readthedocs) format. For now, would you merge the changes unless there is any problem?

**Comment by heitorschueroff** [comment]
> @heitorschueroff I tried to fix the indentation, but I don't know how to modify the docs while conforming to the sphinx (or readthedocs) format. For now, would you merge the changes unless there is any problem?

I'm merging the changes. Thank you for looking into it!

**Pull request closed without merging**